dipper/sources/ClinVar.py

```diff
@@ -962,26 +962,34 @@ def parse():
                             rcv_disease_id = RCV_TraitXRef.get('ID')
                             break
 
+                    # Accept MONDO if no OMIM or Orphanet  # revisit priority
+                    if rcv_disease_db is None or rcv_disease_id is None:
+                        if rcv_disease_db is not None:
+                            break
+                        for RCV_TraitXRef in RCV_Trait.findall('./XRef[@DB="MONDO"]'):
+                            rcv_disease_db = 'MONDO'  # RCV_TraitXRef.get('DB')
+                            rcv_disease_id = RCV_TraitXRef.get('ID')
```

Comment:
IDs already have the MONDO prefix, so I think this will end up double prefixing MONDO:MONDO, RCV_TraitXRef.get('ID')[6:]

Reply:
line 1071?

Reply:
ah i missed this, looks good!